sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/PartitionClosingEventArgs.cs

```diff
@@ -46,6 +46,7 @@ public class PartitionClosingEventArgs
 
             PartitionId = partitionId;
             Reason = reason;
+            CancellationToken = cancellationToken;
```

Comment:
Well.... yeah.... that would seem to be a pretty big oversight on our part.   Thanks!